include/clap/clap.h

```diff
@@ -56,8 +56,10 @@
 #include "ext/draft/file-reference.h"
 #include "ext/draft/midi-mappings.h"
 #include "ext/draft/preset-load.h"
-#include "ext/draft/quick-controls.h"
+#include "ext/draft/remote-controls.h"
 #include "ext/draft/state-context.h"
 #include "ext/draft/surround.h"
 #include "ext/draft/track-info.h"
 #include "ext/draft/tuning.h"
+#include "ext/draft/param-indication.h"
+#include "ext/draft/context-menu.h"
```

Comment:
The rest of the file looks to me as if the includes are sorted alphabetically. Those two includes are out of order.

Reply:
I can sort them